src/components/CategoryChips/CategoryChips.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useCategories } from "context";
+import styles from "./CategoryChips.module.css";
+const CategoryChips = () => {
+  const { categoriesList } = useCategories();
+  return (
+    <div
+      className={`gentle-flex-gap g-flex-align-center ${styles.categories_wrapper}`}
+    >
+      <div className={`pos-relative`}>
+        <input
+          type="radio"
+          name="category-choice"
+          id="All"
+          className={`${styles.category_input}`}
```

Comment:
I like that you are using styles.class approach for className; I learned something new.